GitUI/CommandsDialogs/SettingsDialog/Pages/FormBrowseRepoSettingsPage.cs

```diff
@@ -1,10 +1,24 @@
 ﻿using GitCommands;
+using GitUI.Hotkey;
 using GitUI.Shells;
+using GitUIPluginInterfaces;
+using ResourceManager;
+using ResourceManager.Hotkey;
 
 namespace GitUI.CommandsDialogs.SettingsDialog.Pages
 {
     public partial class FormBrowseRepoSettingsPage : SettingsPageWithHeader
     {
+        private const string _processHistoryUrl = "https://git-extensions-documentation.readthedocs.io/settings.html#process-history-as-tab-otherwise-as-panel";
+        private readonly TranslationString _outputHistoryTooltip
+            = new("""
+                  The output displayed in the process dialog and some trace output is retained and shown in the output history.
+
+                  - With this set, the output history is displayed in a tab in the lower pane of the Browse Repository window.
+                  - With this unset, the output history is displayed in a panel docked to the lower left corner of the Browse Repository window.
+
+                  Focus the output history or toggle the panel visibility using the hotkey {0}.
```

Comment:
Still convinced that could be improved with:
```suggestion
                  Focus (when displayed as a tab) the output history or toggle (when displayed as a panel) the panel visibility using the hotkey {0}.
```

Reply:
Let's find a clearer wording (none of my strengths ;) because the panel is focused, too, when it is shown.